test_scripts/API/SetAudioStreamingIndicator/048_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_media_app_NONE_RPC_HMI_level_not_included.lua

```diff
@@ -0,0 +1,64 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [GeneralResultCodes] DISALLOWED in case app's current HMI Level is not listed in assigned policies
+-- [MOBILE_API] SetAudioStreamingIndicator
+-- [HMI_API] [MOBILE_API] AudioStreamingIndicator enum
+-- [PolicyTable] SetAudioStreamingIndicator RPC
+--
+-- Description:
+-- In case Policy Table doesn't contain current application's HMILevel defined in Policy Table
+-- "functional_groupings" section for a specified RPC,
+-- PoliciesManager must return DISALLOWED resultCode and success:"false" to this RPC requested by the application.
+--
+-- 1. Used preconditions
+-- Allow SetAudioStreamingIndicator RPC by policy for HMI levels: "LIMITED", "BACKGROUND", "FULL"
+-- Register media application. HMI level = "NONE"
+--
+-- 2. Performed steps
+-- Send SetAudioStreamingIndicator(audioStreamingIndicator = "PAUSE")
+--
+-- Expected result:
+-- SDL->mobile: SetAudioStreamingIndicator_response("DISALLOWED", success:false)
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.application1.registerAppInterfaceParams.appHMIType = {"MEDIA"}
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+
+--[[ General Precondition before ATF start ]]
+testCasesForPolicyTable:precondition_updatePolicy_AllowFunctionInHmiLeves({"LIMITED", "BACKGROUND", "FULL"}, "SetAudioStreamingIndicator")
+commonSteps:DeleteLogsFiles()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
```

Comment:
@OVikhrov, "unconsented device" is applicable only for External Proprietary flow. The script is prepared for default SDL build: Proprietary.
Rejected.